Xamarin.Forms.Core/Frame.cs

```diff
@@ -11,6 +11,8 @@ public class Frame : ContentView, IElementConfiguration<Frame>
 
 		public static readonly BindableProperty HasShadowProperty = BindableProperty.Create("HasShadow", typeof(bool), typeof(Frame), true);
 
+		public static readonly BindableProperty CornerRadiusProperty = BindableProperty.Create(nameof(CornerRadius), typeof(float), typeof(Frame), -1.0f);
```

Comment:
`-1` is fine as an internal value. API consumer do not have to know about it's value, they can reset it to default using `frame.ClearValue(CornerRadiusProperty)`